test/e2e_node/util.go

```diff
@@ -408,6 +408,19 @@ func restartKubelet(running bool) {
 
 	stdout, err = exec.Command("sudo", "systemctl", "restart", kubeletServiceName).CombinedOutput()
 	framework.ExpectNoError(err, "Failed to restart kubelet with systemctl: %v, %s", err, string(stdout))
+
+	// wait until the kubelet status succeed
+	gomega.Eventually(func() bool {
+		_, err = exec.Command("sudo", "systemctl", "status", kubeletServiceName).CombinedOutput()
```

Comment:
IIUC about the code [1][2], `CombinedOutput` does not return error as long as the command runs, even if stderror is not empty.  Should we use something like `sudo systemctl is-active kubelet` and assert on the output?

[1] https://cs.opensource.google/go/go/+/refs/tags/go1.17.6:src/os/exec/exec.go;l=557
[2] https://cs.opensource.google/go/go/+/refs/tags/go1.17.6:src/os/exec/exec.go;drc=refs%2Ftags%2Fgo1.17.6;l=337